functors task (1)/student/include/impl.h

```diff
@@ -40,9 +62,22 @@ struct MixingPolicy
      * Whiskey + SevenUp = SevenPlusSeven;
      * Others + Others = Oops;
      */
-    static Cocktail mix(/**???*/)
+    static Cocktail mix(const AlcoholDrink adrink, const NonAlcoholDrink nadrink)
     {
+        if (AlcoholDrink::Gin == adrink && NonAlcoholDrink::LimeJuice == nadrink)
```

Comment:
you can simplify these checks with switch

Reply:
done it